### PR TITLE
Use the etcd image from the registry.suse.de

### PR DESCRIPTION
**Pull request by inercia**

### CODE REVIEW

File: pkg/config/defaults.go [hunk 1]
```diff
@@ -73,6 +73,12 @@ const (
 	DefaultDNSDomain = "cluster.local"
 )
 
+// etcd stuff
```
Comment (MalloZup) [inline pkg/config/defaults.go]: `minor` nitpick:  // etc default values or etcd defaults is better

Comment (MalloZup) [reply pkg/config/defaults.go]: even better // etcd :sunflower: values (with emoticon embedded)

File: pkg/config/defaults.go [hunk 2]
```diff
@@ -73,6 +73,12 @@ const (
...
+const (
+	// the etcd image to use
+	DefaultEtdcImage = "registry.opensuse.org/devel/kubic/containers/container/kubic/etcd:3.3"
```
Comment (sysrich) [inline pkg/config/defaults.go]: I'd recommend leaving a "// FIXME" here, given users shouldn't be automatically guided to use an unofficial, unreviewed, unreleased container, but we don't have an official namespace for an official etcd container here

As kubic-init is under activate development I think this is fine for now, but this certainly needs to be fixed before we start considering it "Beta" or "GA" quality

Comment (inercia) [reply pkg/config/defaults.go]: We are in the process of creating our base images, so we must make sure that image is the offficial, reviewed and released container image for `etcd` for `kubic-init`.

File: pkg/config/defaults.go (hunk 1 above)
Comment (inercia) [reply pkg/config/defaults.go]: Fixed

**Pull request merged**